.github/workflows/unit-tests.yml

```diff
@@ -13,70 +13,72 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  BOOST_VERSION: 1.69.0
+  CMAKE_VERSION: 3.26.0
+  BOOST_VERSION: 1.81.0
```

Comment:
The Boost version of 1.69 corresponds to the minimum boost version we support, see https://github.com/Return-To-The-Roots/s25client/blob/master/CMakeLists.txt#L309

Why the change?

Reply:
1.69.0 does not work anymore on those runners

Reply:
I'll check that. I'm a huge fan of "what isn't tested is broken" hence I'll try to get an older Boost to work on the runners.